backend/src/main/resources/db/migration/internal/V0.290__Update_last_positions_table.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE public.last_positions
+    ALTER COLUMN longitude SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN latitude SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN total_weight_onboard SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN impact_risk_factor SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN probability_risk_factor SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN detectability_risk_factor SET NOT NULL;
+ALTER TABLE public.last_positions
+    ALTER COLUMN risk_factor SET NOT NULL;
```

Comment:
Il y avait `is_at_port` aussi, tu ne le mets pas finalement ?

Reply:
Colonne déjà not null finalement